web/components/FcDrawerViewData.vue

```diff
@@ -357,63 +156,24 @@ export default {
     actionAddLocation() {
       this.setLocationMode(LocationMode.MULTI_EDIT);
     },
-    actionRequestStudy() {
-      this.$router.push({ name: 'requestStudyNew' });
-    },
-    actionShowReportsCollision() {
-      const params = this.locationsRouteParams;
-      this.$router.push({
-        name: 'viewCollisionReportsAtLocation',
-        params,
-      });
-    },
-    actionShowReportsStudy({ category: { studyType } }) {
-      const params = this.locationsRouteParams;
-      this.$router.push({
-        name: 'viewStudyReportsAtLocation',
-        params: {
-          ...params,
-          studyTypeName: studyType.name,
-        },
-      });
+    actionToggleDetailView() {
+      if (this.detailView) {
+        this.setLocationsIndex(-1);
```

Comment:
we'll probably end up doing something similar in View Reports - we can `setLocationIndex(0)` on route enter there, then also `setLocationIndex(i)` whenever a location is selected from the dropdown.